Removes -r from expected "declare" result in bash-rematch test. Makes result consistent with current void-packages bash version.
--- a/test/data/bash-rematch.right
+++ b/test/data/bash-rematch.right
@@ -31,7 +31,7 @@
 8:	    echo "${!i} matches"
  0: $BASH_REMATCH = 456
 +eval typeset -p BASH_REMATCH
-declare -ar BASH_REMATCH=([0]="456")
+declare -a BASH_REMATCH=([0]="456")
 $? is 0
 +c 
 456 matches

